questionnaire/routes.js

```diff
@@ -53,9 +53,13 @@ router
                 const isSummaryPage =
                     formHelper.getSectionContext(response.body.data[0].attributes.sectionId) ===
                     'summary';
-                answers = isSummaryPage
-                    ? await qService.getAnswers(req.cicaSession.questionnaireId)
-                    : answers;
+
+                if (isSummaryPage) {
+                    answers = await qService.getAnswers(req.cicaSession.questionnaireId);
+                    res.header('Cache-Control', 'private, no-cache, no-store, must-revalidate');
+                    res.header('Expires', '-1');
+                    res.header('Pragma', 'no-cache');
```

Comment:
Combine them in to 1 call:
https://expressjs.com/en/api.html#res.set

`res.set()` is aliased as `res.header(field [, value]).`

Reply:
Done.